src/Filters/molecules/FilterInputs.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState, useCallback } from 'react';
+import styled from 'styled-components';
+import BasicSearchInput, { IBasicSearchInput } from '../../Misc/atoms/BasicSearchInput';
+import FilterDropdown, { IFilterDropdown } from '../../Filters/molecules/FilterDropdown';
+import FilterButton from '../../Filters/atoms/FilterButton';
+
+const SearchInputWrapper = styled.div`
+  width: 214px;
```

Comment:
If this is going inside `Container` which uses `flex`, this should be using flex based rules instead of  `width`

Reply:
Updated as we talk with flex 1 1 200px and Wanted to confirm this is the expected result.
Please let me know since this takes the whole space although will resize if more buttons/ searchers are added.
We also talked about a posible max-width, let me know if you have one in mind.
<img width="1337" alt="Screen Shot 2021-09-21 at 17 37 08" src="https://user-images.githubusercontent.com/10409078/134139434-1327fd0c-dd2c-4d26-bf7a-a90acf2765dd.png">